web-src/src/components/MainSidePanel.js

```diff
@@ -132,62 +158,68 @@ export function MainSidePanel(props) {
-        <ul className={style.menu}>
-          <li className={viewType === ViewType.NewSession ? derivedStyle.clickedMenuItem : style.menuItem}>
-            <ClickableImage src={PromptsIcon} width={'20px'} title={formatMessage(intlMessages.mainSidePanel.promptTemplatesMenuItem)} alt={'New prompt template'} onClick={() => setViewType(ViewType.NewSession)} />
-            {mainSidePanel === MainSidePanelType.Expanded
+      { runMode !== RUN_MODE_CF
+        && <Flex direction={'column'} gridArea={'menu'} gap={'size-100'}>
+          <ul className={style.menu}>
+            <li className={viewType === ViewType.NewSession ? derivedStyles.clickedMenuItem : style.menuItem}>
+              <ClickableImage src={PromptsIcon} width={'20px'} title={formatMessage(intlMessages.mainSidePanel.promptTemplatesMenuItem)} alt={formatMessage(intlMessages.mainSidePanel.promptTemplatesMenuItemAltText)} onClick={() => setViewType(ViewType.NewSession)} />
+              {mainSidePanelType === MainSidePanelType.Expanded
               && <Link href="#" UNSAFE_className={style.menuItemLink} onPress={() => setViewType(ViewType.NewSession)}>{formatMessage(intlMessages.mainSidePanel.promptTemplatesMenuItem)}</Link>}
-          </li>
-          <li className={viewType === ViewType.Favorites ? derivedStyle.clickedMenuItem : style.menuItem}>
-            <ClickableImage src={FavoritesIcon} width={'20px'} title={formatMessage(intlMessages.mainSidePanel.favoritesMenuItem)} alt={'Favorites'} onClick={() => setViewType(ViewType.Favorites)} />
-            {mainSidePanel === MainSidePanelType.Expanded
+            </li>
+            <li className={viewType === ViewType.Favorites ? derivedStyles.clickedMenuItem : style.menuItem}>
+              <ClickableImage src={FavoritesIcon} width={'20px'} title={formatMessage(intlMessages.mainSidePanel.favoritesMenuItem)} alt={formatMessage(intlMessages.mainSidePanel.favoritesMenuItemAltText)} onClick={() => setViewType(ViewType.Favorites)} />
+              {mainSidePanelType === MainSidePanelType.Expanded
               && <Link href="#" UNSAFE_className={style.menuItemLink} onPress={() => setViewType(ViewType.Favorites)}>{formatMessage(intlMessages.mainSidePanel.favoritesMenuItem)}</Link>}
-          </li>
-          <li className={style.menuItem}>
-            {mainSidePanel === MainSidePanelType.Expanded
-              ? <Image src={RecentsIcon} width={'20px'} alt={'Recents'} />
-              : <ClickableImage src={RecentsIcon} width={'20px'} title={formatMessage(intlMessages.mainSidePanel.recentsMenuItem)} alt={'Recents'} onClick={() => (sessions?.length > 0) && setMainSidePanelState(MainSidePanelType.Expanded)} />
-            }
-            {mainSidePanel === MainSidePanelType.Expanded
+            </li>
+            <li className={style.menuItem}>
+              {mainSidePanelType === MainSidePanelType.Expanded
+                ? <Image src={RecentsIcon} width={'20px'} alt={'Recents'} />
```

Comment:
Localizable alt text

Reply:
Since this isn't new, I'd open a follow-up ticket to fill the gaps left after the first localisation attempt.